src/java_bytecode/java_class_loader.h

```diff
@@ -23,6 +23,13 @@ Author: Daniel Kroening, kroening@kroening.com
 class java_class_loadert:public messaget
 {
 public:
+  // Default constructor does not use core models
+  // for backward compatibility of unit tests
```

Comment:
1. Try to see if the unit tests pass now when the two fields are set to true
2. If not, add a FIXME here saying that the two fields should be true and the unit tests should be fixed.

Reply:
Changed to default use_core_models=true. Tests pass